client/v2/algod/getTransactionGroupLedgerStateDeltasForRound.go

```diff
@@ -0,0 +1,33 @@
+package algod
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/algorand/go-algorand-sdk/v2/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/v2/types"
+)
+
+// GetTransactionGroupLedgerStateDeltasForRoundParams contains all of the query parameters for url serialization.
+type GetTransactionGroupLedgerStateDeltasForRoundParams struct {
+
+	// Format configures whether the response object is JSON or MessagePack encoded. If
+	// not provided, defaults to JSON.
+	Format string `url:"format,omitempty"`
+}
+
+// GetTransactionGroupLedgerStateDeltasForRound get ledger deltas for transaction
+// groups in a given round.
+type GetTransactionGroupLedgerStateDeltasForRound struct {
+	c *Client
+
+	round uint64
+
+	p GetTransactionGroupLedgerStateDeltasForRoundParams
+}
+
+// Do performs the HTTP request
+func (s *GetTransactionGroupLedgerStateDeltasForRound) Do(ctx context.Context, headers ...*common.Header) (response types.LedgerStateDelta, err error) {
```

Comment:
I'm pretty sure this is the wrong return type. It's defined as:
```
    "TransactionGroupLedgerStateDeltaForRoundResponse": {
      "description": "Response containing all ledger state deltas for transaction groups, with their associated Ids, in a single round.",
      "schema": {
        "type": "array",
        "items": {
          "$ref": "#/definitions/LedgerStateDeltaForTransactionGroup"
        }
      }
    },
```

Reply:
Good point, looking into this

Reply:
This was fixed by adding a top level object into the API spec -- resolving